11_webpack.md

```
@@ -2414,14 +2414,14 @@ We have 2 scripts related to our express server:
 2. `npm run prod` > node runs our server/index.js bundle: build/server-prod-bundle.js
 
 However... We want to preserve practicality, so...
-We are going to create a new script for development in the server side which is going to watch for changes, re-build the bundle and re-start the server. Remember, this is not a dev server... It´s a server which is going to work/operate/execute with our production output letting us work without having to spend too much time in the console.
+We are going to create a new script for development in the server side which is going to watch for changes, re-build the bundle and re-start the server. Remember, this is not a dev server... It's a server which is going to work/operate/execute with our production output letting us work without having to spend too much time in the console.
 For this reason we are going to call it `build:server:flex` to prevent confusions linked to the `dev` reference.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)